integreat_cms/cms/migrations/0098_add_contact.py

```diff
@@ -96,6 +96,7 @@ class Migration(migrations.Migration):
                     "poi",
                     models.ForeignKey(
                         on_delete=django.db.models.deletion.PROTECT,
+                        related_name="contacts",
```

Comment:
I don't think we should be modifying an existing migration, but I really don't know what kind of problems occur if we do it or if we don't do it :sweat_smile: 

Could you test if everything still works if you remove this line? If not it is probably best to create a new migration.